src/css/components/task-file-list.less

```diff
@@ -0,0 +1,22 @@
+/* ==============
+ * Task File List (table)
+ * ==============
+ */
+.task-file-list {
+  margin-top: @base-spacing-unit * 3;
+
+  tr {
+    .btn {
```

Comment:
No, I simply mean to have a reusable class `.reveal` that we can apply on `.btn` that makes the button appear on hover like you're doing here, instead of having to specify this behavior in a one-off way as u've done here.